src/main/clojure/jarl/eval.clj

```diff
@@ -307,9 +308,9 @@
   [source-index key-index value-index block-stmt state]
   (log/debugf "ScanStmt - scanning <%d>" source-index)
   (let [source (state/get-local state source-index)]
-    (if (or (nil? source) (not (coll? source)) (empty? source))
+    (if (or (nil? source) (not (coll? source)))             ; OPA IR docs states 'source' may not be an empty collection; but of we 'break' for such, statements like 'every x in [] { x != x }' will be 'undefined'.
```

Comment:
The nil check looks redudant, as it's covered by `(not (coll? source))`.

Also in the comment: "but of we" -> "but if we"